proposals/gc/Overview.md

```diff
@@ -275,7 +278,7 @@ Structures can be compared for identity:
 ```
 (same (new $point ...) (new $point ...))  ;; false
 ```
-TODO: Could even allow heterogeneous equality (equality between operands of different type), but that might lead to some discontinueties or even prevent some potential optimizations?
+TODO: Could even allow equality comparisons between operands of different type, but that might lead to some discontinuities or even prevent some potential optimizations?
```

Comment:
Well, it's concise and convenient to use in discussions, so introducing it should help.